pkg/metrics/bootstrap.go

```diff
@@ -0,0 +1,62 @@
+package metrics
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel/codes"
+
+	v1 "github.com/jaegertracing/jaeger-operator/pkg/apis/jaegertracing/v1"
+	"github.com/jaegertracing/jaeger-operator/pkg/tracing"
+
+	prometheusclient "github.com/prometheus/client_golang/prometheus"
+	log "github.com/sirupsen/logrus"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/metric/prometheus"
+	"go.opentelemetry.io/otel/metric/global"
+	export "go.opentelemetry.io/otel/sdk/export/metric"
+	"go.opentelemetry.io/otel/sdk/metric/aggregator/histogram"
+	controller "go.opentelemetry.io/otel/sdk/metric/controller/basic"
+	processor "go.opentelemetry.io/otel/sdk/metric/processor/basic"
+	selector "go.opentelemetry.io/otel/sdk/metric/selector/simple"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+const meterName = "jaegertracing.io/jaeger"
+
+// Bootstrap prepares a new tracer to be used by the operator
+func Bootstrap(ctx context.Context, namespace string, client client.Client) {
+	tracer := otel.GetTracerProvider().Tracer(v1.CustomMetricsTracer)
+	ctx, span := tracer.Start(ctx, "bootstrap")
```

Comment:
is this span recording events/ reported? 

If the bootstrap creates tracer than the span might be noop/default if it is created before initialization.

Reply:
The tracer is already created when this bootstraping is called, this bootstraps the metrics for Open Telemetry,